joomla/joomla-cms
# [#31139] Two functions in com_content article model need to use JDatabase

Fixing issue from tracker:
http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_item_id=31139

https://github.com/joomla/joomla-cms/blob/master/components/com_content/models/article.php#L268

Look at the hits and storeVote functions in the article model (frontend) of
com_content. We need to convert these to use JDatabase

This issue seemed confirmed to me and easy to fix, so I made the changes to
those methods. I'm not sure if my code is well written but everything seemed to
work as normal after my changes.
Can anyone else please test and check if the code is correct?

Test instructions:
1. (hits): Enter a random article page, and refresh it to see the hits counter
incrementing;
2. (storeVote): Also inside an article page, rate the article from 1 to 5. You
may need to enable the 'Show voting' option, inside Article Manager's Options,
to view the rate option.
